CMakeLists.txt

```diff
@@ -88,8 +88,8 @@ gz_find_package (Qt5
 
 #--------------------------------------
 # Find gz-sim
-gz_find_package(gz-sim9 REQUIRED PRIVATE COMPONENTS gui)
-set(GZ_SIM_VER ${gz-sim9_VERSION_MAJOR})
+gz_find_package(gz-sim REQUIRED PRIVATE COMPONENTS gui)
+set(GZ_SIM_VER ${gz-sim_VERSION_MAJOR})
```

Comment:
we could alternatively delete the `GZ_SIM_VER` variable since I believe it is not used after these changes

Reply:
removed in https://github.com/gazebosim/gz-launch/pull/292/commits/28765fe586bd72b21f3556b2fdc765e6dcbae710